android/src/main/java/com/hieuvp/fingerprint/ReactNativeFingerprintScannerModule.java

```diff
@@ -287,13 +285,17 @@ private void legacyAuthenticate(final Promise promise) {
             @Override
             public void onSucceed() {
                 promise.resolve(true);
-                ReactNativeFingerprintScannerModule.this.release();
             }
 
             @Override
             public void onNotMatch(int availableTimes) {
-                mReactContext.getJSModule(RCTDeviceEventEmitter.class)
-                    .emit("FINGERPRINT_SCANNER_AUTHENTICATION", "AuthenticationNotMatch");
+                if( availableTimes <= 0 ){
+                    mReactContext.getJSModule(RCTDeviceEventEmitter.class)
+                        .emit("FINGERPRINT_SCANNER_AUTHENTICATION", "AuthenticationLockout");
+                }else{
```

Comment:
```suggestion
                if (availableTimes <= 0) {
                    mReactContext.getJSModule(RCTDeviceEventEmitter.class)
                        .emit("FINGERPRINT_SCANNER_AUTHENTICATION", "AuthenticationLockout");
                } else {
```

I think general whitespace style (the majority at least) is like this in the file